### ISSUE TITLE
feat: [Youtube] Make automatically turning on the app from websites possible

### ISSUE THREAD

**Issue by alan23273850** [ReVanced/revanced-patches#2777]
### Feature description

To make this feature possible, two kinds (or four if you see from the official app) of urls listed in the following figure should be validated in this app. Can the developers help us to do this? (for now only "0" urls have been validated)
![426720558_394806406526155_8051940403014583803_n](https://github.com/ReVanced/revanced-patches/assets/10044077/c11367a1-c454-47e8-9e4a-f6b964efb339)

### Motivation

The official app has this feature, but the patched app does not.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by alan23273850** [comment]
but my Google Chrome still opens the link with the disabled official youtube app, so I will find another solution and thanks!

**Comment by alan23273850** [comment]
I've followed https://www.xda-developers.com/uninstall-carrier-oem-bloatware-without-root-access/ and successfully uninstalled the stock YouTube. The process is quite easy. But if I click the link, then the screen goes to the official play store instead of revanced. Is there another solution? Thanks!

**Comment by alan23273850** [comment]
Is microG or mindetach required? My OS said the installation of microG failed, but I can still use ReVanced as usual...

**Comment by alan23273850** [comment]
https://xiaomi.eu/community/threads/revanced-not-opening-from-chrome-on-latest-xiaomi-eu-stable.69453/

According to the above post, the problem only occurs in Chrome. So finally, I decided to copy and paste the video link into the Google search bar and then pressed enter, it will redirect to YouTube Revanced (Extended) automatically.

The post said the problem has been solved with YouTube Revanced "Extended", but my problem still occurs. So strange!
